Week3/js-exercises/ex5-LotteryMachine.js

```diff
@@ -27,11 +27,37 @@ Don't you just love the thrill of the lottery? What if I told you we can make ou
 */
 
 function threeFive(startIndex, stopIndex, threeCallback, fiveCallback) {
-  const numbers = [];
   // make array
+  const numbers = [startIndex];
+  for ( let i = 0; i <= stopIndex; i++){
+
+  if(startIndex !== stopIndex){
+    startIndex ++
+    numbers.push(startIndex) 
+  }
+  }
```

Comment:
This surely creates the array. Consider the following and decide for yourself what you prefer and why.
```suggestion
  const numbers = [];
  for ( let i = startIndex; i <= stopIndex; i++){
    numbers.push(i) 
  }
```